.doc_gen/metadata/medical-imaging_metadata.yaml

```diff
@@ -994,7 +1001,7 @@ medical-imaging_Scenario_TaggingImageSets:
 medical-imaging_Scenario_ImageSetsAndFrames:
   title: Get started with &AHI; image sets and image frames using an &AWS; SDK
   title_abbrev: Get started with image sets and image frames
-  synopsis: get started with &AHI; image sets and image frames.
+  synopsis: import DICOM files and download image frames in &AHI;.<para>The implementation is structured as a workflow command-line application.</para>
```

Comment:
I think these `<para>` tags might not work right. If I remember correctly, the whole synopsis is wrapped in `<para></para>`, so I think what you want is `</para><para>` like `...&AMI;.</para><para>The implementation...` and remove the `</para>` at the end.

Reply:
The original version builds and renders fine, so we can keep this as is.